package.json

```diff
@@ -65,21 +69,20 @@
     "parcel-bundler": "^1.12.4",
     "prettier": "1.17.1",
     "sinon": "^7.5.0",
-    "sinon-chai": "^3.5.0"
+    "sinon-chai": "^3.5.0",
+    "ts-mocha": "^7.0.0",
+    "ts-node": "^8.9.0",
+    "typescript": "^3.8.3"
   },
   "husky": {
     "hooks": {
       "pre-commit": "lint-staged"
     }
   },
   "lint-staged": {
-    "*.js": [
-      "prettier --write",
-      "eslint --fix",
-      "git add"
-    ],
     "*.ts": [
       "prettier --write",
+      "eslint --fix",
```

Comment:
Maybe we want to consider moving to tslint?

Reply:
Tslint is deprecated and they recommend using eslint with the typescript plugin